matrix-sdk/src/main/java/org/matrix/androidsdk/util/PermalinkUtils.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2018 New Vector Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.matrix.androidsdk.util;
+
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.text.TextUtils;
+
+import org.matrix.androidsdk.rest.model.Event;
+
+/**
+ * Useful methods to create permalink
+ */
+public class PermalinkUtils {
```

Comment:
Maybe we should avoid adding more Utils classes and using static methods?
I do prefer having something as a PermalinkFactory we can  inject and mock in Unit Tests